todo_app/app.py

```diff
@@ -3,47 +3,70 @@
 from todo_app.flask_config import Config
 from todo_app.data.session_items import get_items, get_item, add_item
 from todo_app.data.trello_items import TrelloService
+from todo_app.data.view_model import ViewModel
 
-app = Flask(__name__)
-app.config.from_object(Config())
-
-# Create an instance of the TrelloService class with the API key and API token
-api_key = os.getenv('API_KEY')
-api_token = os.getenv('API_TOKEN')
-trello_service = TrelloService(api_key, api_token)
-
-# Update the function that handles the GET request for fetching todo items
-@app.route('/')
-def index():
-    # Get Board ID and List ID from env file
-    list_id = os.getenv('TO_DO_LIST_ID')
-    # Use the TrelloService class method to fetch todo items
-    todo_items = trello_service.fetch_todo_items(list_id)
-    
-    return render_template('index.html', items=todo_items)
-
-@app.route('/add', methods=['POST'])
-def add_new_card():
-    # Get List ID from env file
-    list_id = os.getenv('TO_DO_LIST_ID')
-    card_name = request.form.get('item')
-    updated_items = add_item(card_name)
-    # Call the TrelloService class method to create a new card
-    card_id = trello_service.create_todo_card(list_id, card_name)
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
+
+
+    # Create an instance of the TrelloService class with the API key and API token
+    api_key = os.getenv('API_KEY')
+    api_token = os.getenv('API_TOKEN')
+    trello_service = TrelloService(api_key, api_token)
+
+    # Update the function that handles the GET request for fetching todo items
+    @app.route('/')
+    def index():
+        # Get Board ID and List ID from env file
+        to_do_list_id = os.getenv('TO_DO_LIST_ID')
+        doing_list_id = os.getenv('DOING_LIST_ID')
+        done_list_id = os.getenv('DONE_LIST_ID')
+        
+        # Use the TrelloService class method to fetch items by status
+        todo_items = trello_service.fetch_todo_items(to_do_list_id)
+        doing_items = trello_service.fetch_todo_items(doing_list_id)
+        done_items = trello_service.fetch_todo_items(done_list_id)
+        
+        # Create ViewModel instances for each status category
+        todo_view_model = ViewModel(todo_items)
+        doing_view_model = ViewModel(doing_items)
+        done_view_model = ViewModel(done_items)
```

Comment:
This works, but is a sign that our ViewModel properties aren't actually doing anything useful (and in fact, the properties don't actually work with your code as it currently operates - more below!) since _each_ ViewModel is being constrained to only have one type of card, but the point of the properties that you unit tested were to filter the cards

Even if we're getting them from three different endpoints as here, it would be nicer to put them all together for the view model:
```python
        view_model = ViewModel(todo_items + doing_items + done_items)
```